dhcp-relay/dhcp_kern_xdp.c

```diff
@@ -326,11 +426,11 @@ int xdp_dhcp_relay(struct xdp_md *ctx) {
 	bpf_printk("Parsing DHCP packet, opcode %i, hops %i", dhcp->op, dhcp->hops);
 
 	if (dhcp->op == DHCP_REQUEST && (eth->h_dest[0] == 0xff
-		&& eth->h_dest[1] == 0xff
-		&& eth->h_dest[2] == 0xff
-		&& eth->h_dest[3] == 0xff
-		&& eth->h_dest[4] == 0xff
-		&& eth->h_dest[5] == 0xff)) {
+			&& eth->h_dest[1] == 0xff
+			&& eth->h_dest[2] == 0xff
+			&& eth->h_dest[3] == 0xff
+			&& eth->h_dest[4] == 0xff
+			&& eth->h_dest[5] == 0xff)) {
```

Comment:
This calls for an `is_ether_broadcast()` helper (which can then be inverted below)